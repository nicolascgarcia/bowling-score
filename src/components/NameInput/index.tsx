import styled from 'styled-components';

export default styled.input`
	width: 100%;
	margin-right: 8px;
	border-top-width: 0;
	border-left-width: 0;
	border-right-width: 0;
	border-color: #a1a1a1;
	border-bottom-width: 2px;
	background-color: #121214;
	color: #fff;
	font-size: 26px;
	font-weight: 'bold';
	font-family: 'Montserrat', sans-serif;

	:focus-visible {
		outline: none;
	}
`;
