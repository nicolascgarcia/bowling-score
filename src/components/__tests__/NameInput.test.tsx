/* eslint-disable import/no-extraneous-dependencies */
import renderer from 'react-test-renderer';
import NameInput from '../NameInput';

describe('<NameInput/>', () => {
	it('should render NameInput', () => {
		const component = renderer.create(<NameInput />);
		const tree = component.toJSON();
		expect(tree).toMatchSnapshot();
	});
});
