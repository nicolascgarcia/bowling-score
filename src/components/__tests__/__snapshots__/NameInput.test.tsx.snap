// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<NameInput/> should render NameInput 1`] = `
<input
  className="sc-bczRLJ gBGFYF"
/>
`;
